server/server.go

```diff
@@ -3,28 +3,62 @@ package server
 import (
 	"context"
 	"fmt"
+
 	pb "github.com/rode/rode/proto/v1alpha1"
-	grafeas "github.com/rode/rode/protodeps/grafeas/proto/v1beta1/grafeas_go_proto"
+	grafeas_proto "github.com/rode/rode/protodeps/grafeas/proto/v1beta1/grafeas_go_proto"
+	grafeas_project_proto "github.com/rode/rode/protodeps/grafeas/proto/v1beta1/project_go_proto"
 
 	"github.com/golang/protobuf/proto"
 	"go.uber.org/zap"
+	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/encoding/protojson"
 )
 
+// NewGrafeasClients construct for GrafeasClients
+func NewGrafeasClients(grafeasEndpoint string) (*GrafeasClients, error) {
+	connection, err := grpc.Dial(grafeasEndpoint, grpc.WithInsecure())
```

Comment:
is there a reason that the `DialContext` change I made in #17 was removed here?

Reply:
Nope, missed it when I rebased the changes. I added it back.